ingest.py

```diff
@@ -1,316 +0,0 @@
-        return None
-
-    @staticmethod
-    def _plusminus_conversion(text):
-        """
-        Converts the text to number value, with a default of 0.
-
-        :param text: String to convert, one of r"b+" or "".
-        :returns: Float casted number.
-        """
-        return float(text) if text else 0
-
-    def find(self):
-        """
-        Adds all gamelogs from a basketball-reference url to the database.
-        """
-        page = requests.get(self._url, params=self._payload).text
-        soup = BeautifulSoup(page)
-        regular_table = soup.find('table', {'id': self._regular_id})
-        playoff_table = soup.find('table', {'id': self._playoff_id})
-        if regular_table:
-            header_add = self._get_header_add(regular_table)
-        else:
-            header_add = self._get_header_add(playoff_table)
-        if header_add:
-            header = self._create_header(header_add)
-            if regular_table:
-                self._table_to_db('regular', regular_table, header)
-            if playoff_table:
-                self._table_to_db('playoff', playoff_table, header)
-
-    @staticmethod
-    def _get_header_add(table):
-        """Finds and returns the header of a table.
-
-        :param table: A basketball-reference stats table.
-        :returns: Header of the table as a list of strings.
-        """
-        def replace_titles(title):
-            return multiple_replace(
-                title,
-                {'%': 'P', '3': 'T', '+/-': 'PlusMinus'})
-        titles = (
-            replace_titles(str(th.get_text()))
-            for th in table.find_all('th')
-        )
-        return list(unique_everseen(titles))
-
-    def _create_header(self, header_add):
-        """
-        Creates the header including the initial header and the header_add.
-
-        :param header_add: List of strings found from the table to add to the
-            initial_header.
-        """
-        header = self._initial_header + header_add
-        header[9] = 'Home'
-        header.insert(11, 'WinLoss')
-        return without(header, 'FGP', 'FTP', 'TPP')
-
-    def _table_to_db(self, season, table, header):
-        """
-        Adds all gamelogs in a table to the database.
-
-        :param season: The season type of the gamelog, either 'playoff' or
-            'regular'.
-        :param table: A basketball-reference stats table.
-        :param header: Header of the table.
-        """
-        gamelogs = []
-        for row in table.find_all('tr'):
-            stat_values_add = self._stat_values_parser(row.find_all('td'))
-            # Skip headers which are found throughout the table.
-            if not stat_values_add:
-                continue
-            stat_values = (
-                self._initial_stat_values +
-                [season] +
-                stat_values_add
-            )
-            gamelogs.append(dict(izip(header, stat_values)))
-        self._insert_gamelogs(gamelogs)
```

Comment:
I usually define a method in the class that raises NotImplementedError, instead of not defining the method at all. I know there are no "public" methods in this class, but it makes it clear what you have to do when you inherit from this.

The fact that this is an abstract class should probably go in a class docstring as well.